### ISSUE TITLE
Movie Set artwork is not found

### ISSUE THREAD

**Issue by asm0dey** [Komet/MediaElch#1746]
**Describe the bug**
At least in my case the artwork for movie sets cannot be found

**To Reproduce**

Have several movies in one set, for example "Ice Age Collection"
Navigate to the set
Click on the empty image at the poster, observe a dialogue like this
![image](https://github.com/Komet/MediaElch/assets/1297282/8c672220-5815-4371-a4b2-e23aafcb7a1d)

Check if the collection exists on fanart.tv: https://fanart.tv/movie/8354/ice-age-collection/

**Expected behavior**
File downloads automatically

**MediaElch Version:**
 - [ ] 2.10.7-dev (nightly)
 - [x] 2.10.6 (stable)

**Operating System:**
 - [ ] Windows
 - [ ] macOS
 - [x] Linux (distribution: Arch)

**Comment by bugwelle** [comment]
Indeed. As of now, you need to search for a _movie_, i.e. remove the "Collection" suffix.
